README.md

```diff
@@ -8,15 +8,17 @@ However, we are committed to continuing our mission to democratize investment re
 
 To achieve that, we are working on a new CLI tool built on top of the OpenBB Platform that will soon be available to everyone for free.
 
+[![GitHub release](https://img.shields.io/github/release/OpenBB-finance/OpenBBTerminal.svg?maxAge=3600)](https://github.com/OpenBB-finance/OpenBBTerminal/releases)
+
 ---
+
```

Comment:
Should this image be replaced?, OpenBB Terminal logo?

Reply:
I tend to agree, buttt I don't know which to use!
@IgorWounds ?

Reply:
Maybe we can change the image when we decide to rename the repo. 
cc @jmaslek

Reply:
image changed